examples/resources/istio/service-entries.yaml

```diff
@@ -0,0 +1,240 @@
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: serviceentries
+  namespace: lyczeq
```

Comment:
I think namespace should be `kube-public`

Reply:
Yes, definitely :D